tests/boards/intel_adsp/ssp/Kconfig

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright (c) 2022, Intel
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+mainmenu "Dai SSP Test"
+
+source "Kconfig.zephyr"
+
+config INTEL_MN
```

Comment:
wondering, what happens if this isn't selected? Can SSPs still be used properly with all the sampling rates?